pingpong/msg_pingpong.c

```diff
@@ -122,31 +136,37 @@ static int sync_test(void)
 	}
 	credits = max_credits;
 
-	ret = opts.dst_addr ? send_xfer(16) : recv_xfer(16);
+	/* In order to not be included in the data integrity check, pass iter=-1 */
+	ret = opts.dst_addr ? send_xfer(16, -1) : recv_xfer(16, -1);
 	if (ret) {
 		return ret;
 	}
 
-	return opts.dst_addr ? recv_xfer(16) : send_xfer(16);
+	return opts.dst_addr ? recv_xfer(16, -1) : send_xfer(16, -1);
 }
 
 static int run_test()
 {
 	int ret, i;
 
+	if (!verify_data)
+                recv_buf_size = buffer_size;
+	else
+		recv_buf_size = opts.transfer_size;
```

Comment:
The spacing is off here, but this change doesn't make sense.  Why should the size of the receive buffer depend on whether or not data verification is on?